sql/core/src/test/java/test/org/apache/spark/sql/JavaDatasetSuite.java

```diff
@@ -1711,6 +1711,23 @@ public void testEmptyBean() {
     Assert.assertEquals(1, df.collectAsList().size());
   }
 
+  public static class ReadOnlyPropertyBean implements Serializable {
+      public boolean isEmpty() {
```

Comment:
We use 2-space indentation. Could you fix the style, @gbloisi-openaire ?

Reply:
I just committed the required changes, also for [PR-42634](https://github.com/apache/spark/pull/42829) that is the same patch against master and 3.5.

Reply:
Thank you for updated. Ya, I know that the original PR is merged already.